Shared/ThresholdKeyView.swift

```diff
@@ -16,9 +16,13 @@ struct ThresholdKeyView: View {
     @State private var resetAccount = true
     @State var threshold_key: ThresholdKey!
     @State var shareCount = 0
-
+    @State private var isLoadingAddPassword = false
+    @State private var isLoadingChangePassword = false
```

Comment:
This is not incorrect, though I think we can reduce the amount of variables we need in the app state.

Could we not just have a `showLoadingSpinner` variable for all cases where it is needed?

Reply:
Done